fashify/src/utils/utils.js

```diff
@@ -54,11 +54,17 @@ const sortByPrice = (data, sortCat) => {
 const priceRangeFilter = (data, maxValue) => {
   return data.filter((el) => Number(el.price) <= maxValue);
 };
-
+const searchFilter = (data, keyword) => {
+  if (keyword === "") return data;
+  return data.filter((el) =>
+    el.title.toLowerCase().startsWith(keyword.toLowerCase())
+  );
+};
```

Comment:
Instead of startsWith you can use includes function, then it will search in whole title

Reply:
I actually used that but putting men it was not discarding women . So I used startWith here